app/javascript/order_screen.js

```
@@ -354,7 +365,8 @@ document.addEventListener('turbolinks:load', () => {
             }
 
             /* eslint-disable no-undef */
-            bootstrap.Modal.getOrCreateInstance('#credit-mutation-modal').hide();
+          bootstrap.Modal.getOrCreateInstance('#credit-mutation-modal').hide();
```

ESLint

[error] Expected indentation of 12 spaces but found 10.

(indent)